[0m[[0m[31merror[0m] [0m[0m/home/adeye/Documents/EPFL/MA2/Systems/Assignment/task2/src/main/scala/Task2_random.scala:6:8: Task2 is already defined as object Task2[0m
[0m[[0m[31merror[0m] [0m[0mobject Task2 {[0m
[0m[[0m[31merror[0m] [0m[0m       ^[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
